### ISSUE TITLE
link overrides dont work

### ISSUE THREAD

**Issue by 89z** [NLKNguyen/papercolor-theme#177]
In another issue, someone suggested I could use a command like this to override a color:

~~~vim
hi link goPredefinedIdentifiers Keyword
~~~

in my `vimrc` file. However I noticed that if I also have PaperColor enabled, then any command like that is ignored.

**Comment by ug02fast** [comment]
call `hi link goPredefinedIdentifiers Keyword` only after the `syntax on` in your `.vimrc`

you might need the exclamation mark to override defaults too `hi!...`

**Comment by 89z** [comment]
> call `hi link goPredefinedIdentifiers Keyword` only after the `syntax on` in your `.vimrc`

@ug02fast thanks for the comment, but I dont have `syntax on` in my `.vimrc` file.

**Comment by ug02fast** [comment]
I had the same issue as you and this is what got it working for me. 
![image](https://user-images.githubusercontent.com/5507707/160028265-7a55228a-af67-4f4f-a21d-bad0b711cc6e.png)

**Comment by 89z** [comment]
@ug02fast thanks for the comment, but I dont have `syntax on` in my `.vimrc` file.

**Comment by ug02fast** [comment]
Syntax on is what enables highlighting so you will need it unless I'm missing something.

**Comment by 89z** [comment]
I dont think thats true, as I dont have it in my `.vimrc`, and I still have highlighting.

**Comment by ug02fast** [comment]
It's possible that one of your plugins turns it on.

**Comment by 89z** [comment]
I dont use plugins...
